src/lib/utils/searchBar.ts

```diff
@@ -0,0 +1,20 @@
+import { FungibleToken, GenieCollection } from 'nft/types'
```

Comment:
nit: would bet better if this could live within the NavBar component dir imo

Reply:
We like to have all datatypes in the `types` folder for organization. GenieCollection is used in several places outside of the NavBar